internal/db/db.go

```diff
@@ -86,44 +93,18 @@ type Path struct {
 }
 
 type Content struct {
+	Kind  string `json:"kind"`
 	Slice string `json:"slice"`
 	Path  string `json:"path"`
 }
 
-type dbBase struct {
-	Kind string `json:"kind"`
-}
-
-type dbPackage struct {
-	dbBase
-	Package
-}
-
-type dbSlice struct {
-	dbBase
-	Slice
-}
-
-type dbPath struct {
-	dbBase
-	Path
-}
-
-type dbContent struct {
-	dbBase
-	Content
-}
-
 // AddPackage adds a "package"-kind entry to the DB.
 func (dbw *DBWriter) AddPackage(pkg *Package) error {
```

Comment:
From what we discussed this function now is a little redundant, right? The caller has to:
```go
err := dbw.AddPackage(db.Package{...})
```
instead of:
```
err := jsonwall.Add(db.Package{...})
```

Can we please remove it and interact with the struct and the functions directly?

Reply:
Removed! And did a refactor too -> https://github.com/letFunny/chisel/pull/11/commits/c129fea2631fb31fe6ce26dd7be3f02bbcb22434.